packages/console/src/containers/AppContent/components/Contact/ContactModal/ContactIcon.tsx

```diff
@@ -0,0 +1,21 @@
+import type { SVGProps } from 'react';
+
+function ContactIcon(props: SVGProps<SVGSVGElement>) {
```

Comment:
@charIeszhao, should we use the SVG transformer for this icon?

Reply:
Yes, we should.

Reply:
But this is a legacy issue. Previously when we introduce the svg transformer plugin, we forgot to process those tsx svg components in the sidebar folder... And this becomes a problem in this PR